src/widget.tsx

```diff
@@ -55,9 +66,22 @@ export class JupyterCadPanel extends ReactWidget {
     super.dispose();
   }
 
+  getView(key: string): JSONValue | undefined {
+    return this._view.get(key);
+  }
+
+  setView(key: string, value: JSONValue): void {
```

Comment:
Can we enforce the typing of `key` here to `'axes'|...` instead of a generic `string`?

Reply:
The idea was to use this API for any helper | display related attributes.

Reply:
I see, but the downstream needs to put the correct key to activate the related function. So I think it's safer to limit the scope of the `key`. We will enlarge this interface when others helpers/settings come.

Reply:
Thanks Carlos!